### DISCUSSION TITLE
About the recordings for the workshop

### DISCUSSION THREAD

**Discussion by acf986** [ros-industrial-consortium/scan_n_plan_workshop#30] [Q&A]
Hi, sorry to bother. 
As the scan_n_plan workshop is part of the ROSWorld 2021 workshop, 
is it possible to consider to link the recording of the virtual workshop?

**Comment by marip8** [comment]
I'm not sure if the workshop was actually recorded, and if it was, whether that recording was ever released to the public. I did a somewhat detailed explanation of how it works in a ROS-I community meeting a few months back. Here is the [link to the video](https://youtu.be/mkYRL72wiPo). Hopefully this provides enough information to get started (and is more up-to-date than the ROSCon '21) version.
